configs/components/oifs/oifs.yaml

```diff
@@ -998,8 +998,8 @@ choose_major_version:
                 add_namelist_changes:
                         fort.4:
                                 NAMRES:
-                                        NFRRES: ${nfrres}
-                                        NRESTS${nrest_ind}: ${nfrests}
+                                        NFRRES: 1
+                                        NRESTS${:}: -1 ${nfrests}
```

Comment:
I don't think this syntax could possibly work. What do you want to do exactly? At the moment is trying to find a variable named `:`